tools/csv-merger/csv-merger.go

```diff
@@ -12,9 +12,10 @@ import (
 	"strings"
 	"text/template"
 
-	"github.com/coreos/go-semver/semver"
+	"github.com/blang/semver"
```

Comment:
Why this change?

Reply:
I had to update the operator-lifecycle-manager package we use because of a bug they had when I posted OperatorGroup objects.

For some reason they switched the semver package, which called csv-merger to fail to compile until I updated our code to reflect the change as well.